.travis.yml

```diff
@@ -21,7 +21,24 @@ before_script:
     fi
 script: "node ./scripts/build.js"
 after_success:
-  - if [ $IS_PUSH_TO_MASTER ]; then
-      node ./scripts/build-docs.js;
+  - if [ $IS_PUSH_TO_MASTER && "${TRAVIS_JOB_NUMBER}" == *1 ]; then
```

Comment:
Checks to see if the job number ends with `1` - meaning it is the very first job. Travis job numbers look like `2349234.1` where `2349234` is the build number and `1` is the job within that build.